sound/soc/soc-topology.c

```diff
@@ -2094,6 +2107,12 @@ static int soc_tplg_link_config(struct soc_tplg *tplg,
 		return ret;
 	}
 
+	/* add it to dobj_list so we can unload it in _remove */
```

Comment:
unload it in snd_soc_tplg_component_remove, be explicit to avoid confusions with remove_link().